test/views/object.spec.js

```diff
@@ -0,0 +1,40 @@
+import {expect, shallowRender} from '../test-helpers'
+import React from 'react'
+
+import {parse} from '../../app/scripts/utils/path'
+import ObjectView from '../../app/scripts/views/object'
+
+import Links from '../../app/scripts/views/object/links'
+import DisplayData from '../../app/scripts/views/object/display-data'
+import PermaLink from '../../app/scripts/views/object/perma-link'
+
+describe('Object', () => {
+  it('renders', () => {
+    const path = parse('/ipfs/hello/world')
+    const permalink = parse('/perma/hello/world')
+    const el = shallowRender(
+      <ObjectView
+          path={path}
```

Comment:
Generally, I try to alphabetize attributes in newline-delimited lists like these. It makes lookup much easier. Just a thought.